backend/src/gamestate/game_repo.rs

```diff
@@ -1,37 +1,36 @@
 use crate::gamestate::game::Game;
-use actix_web::error::ErrorBadRequest;
 use anyhow::anyhow;
 use async_trait::async_trait;
 use jwt_simple::prelude::*;
 use nanoid::nanoid;
-use std::fmt::Error;
-use std::sync::Arc;
 
 #[async_trait]
 pub trait GameRepo {
-    async fn create_game(&mut self, name: String) -> anyhow::Result<(Game)>;
+    async fn create_game(&mut self, name: String) -> anyhow::Result<Game>;
 }
 
 #[derive(Clone)]
-pub struct PostgresGameRepo {
+pub struct StableGameRepo {
     games: Vec<Game>,
 }
 
-impl PostgresGameRepo {
+impl StableGameRepo {
     pub fn new(games: Vec<Game>) -> Self {
         Self { games }
     }
 }
 
 #[async_trait]
-impl GameRepo for PostgresGameRepo {
-    async fn create_game(&mut self, name: String) -> anyhow::Result<(Game)> {
```

Comment:
Took me a while to understand that this was the game's author's name, not the game's name (which it obviously doesn't have, but still)

Reply:
Resolved